tests/testthat/test-git.R

```diff
@@ -1,4 +1,6 @@
 without_internet({
+  scoped_experimental_silence()
```

Comment:
These are unrelated to this PR and 'just' quieting down tests? Or something else?

Reply:
Correct, these test files were added in #264, and I didn't notice them before because I was iterating inside of a live R session and these warnings only fire the first time.